packages/harpguru-core/src/components/option-list/option-list.tsx

```diff
@@ -0,0 +1,68 @@
+import Animated from 'react-native-reanimated'
+import { FlatList } from 'react-native-gesture-handler'
+import { StyleSheet, View, SafeAreaView } from 'react-native'
+import React from 'react'
+
+import { getOptionSizes } from '../../utils'
+import type { WithTransition } from '../../types'
+import { getWindowDimensions } from '../../packages/get-window-dimensions'
+
+export type OptionListProps = WithTransition & {
+  readonly useItems: () => ReadonlyArray<React.ReactElement>
+  readonly twoColumns?: boolean
+  readonly useLeftColumnLabel?: () => React.ReactElement
```

Comment:
Need to play with the type system to see whether it's possible to enforce that this is a `OptionLabel` component, and nothing else.